samplesheets/tests/test_permissions_api_taskflow.py

```diff
@@ -74,3 +81,489 @@ def test_get_archive(self):
         ]
         self.assert_response_api(url, good_users, 200, data=self.post_data)
         self.assert_response_api(url, self.anonymous, 401, data=self.post_data)
+
+
+class TestIrodsRequestAPIViewBase(
+    SampleSheetIOMixin, SampleSheetTaskflowMixin, TaskflowAPIPermissionTestBase
+):
+    """Base test class for IrodsRequestAPIView tests"""
```

Comment:
This should probably read `Base test class for IrodsRequestAPIView permission tests`.